extensions/blocks/slideshow/slideshow.php

```diff
@@ -24,5 +24,161 @@
  */
 function jetpack_slideshow_block_load_assets( $attr, $content ) {
 	Jetpack_Gutenberg::load_assets_as_required( 'slideshow' );
+	if ( class_exists( 'Jetpack_AMP_Support' ) && Jetpack_AMP_Support::is_amp_request() ) {
+		return jetpack_slideshow_block_render_amp( $attr );
+	}
 	return $content;
 }
+
+/**
+ * Render slideshow block for AMP
+ *
+ * @param array $attr Array containing the slideshow block attributes.
+ *
+ * @return string
+ */
+function jetpack_slideshow_block_render_amp( $attr ) {
+	static $wp_block_jetpack_slideshow_id = 0;
+	$wp_block_jetpack_slideshow_id++;
+
+	$ids      = empty( $attr['ids'] ) ? array() : $attr['ids'];
+	$autoplay = empty( $attr['autoplay'] ) ? false : $attr['autoplay'];
+	$classes  = array(
```

Comment:
Should we rely on `Jetpack_Gutenberg::block_classes()` here?

Reply:
Yes, good catch. `block_classes()` was introduced after this PR was first opened, so the old approach was still in place. https://github.com/Automattic/jetpack/pull/13009/commits/15ab41803d51ec0d3f2f186ee956c3deccfd6dd6